chucks-pick3-web/angular-app/src/app/components/info-requirements.component/info-requirements.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="how-to-header" id="requirements">
   <h1><b>Requirements:</b></h1>
-    </div>
-    <div class="what-you-need">
-  <img src="/assets/images/Info page -11.png" alt="IPhone">
+</div>
+<div class="what-you-need">
+  <img src="src/app/assets/images/Info page -11.png" alt="IPhone">
```

Comment:
Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

Reply:
reverted back to assets/images.